features/fixtures/delayed_job/app/app/models/test_model.rb

```diff
@@ -0,0 +1,13 @@
+class TestModel < ApplicationRecord
+  def self.print
```

Comment:
It might be preferable to avoid method names like `print` and `fail` as they override `Kernel#print` and `Kernel#fail`